.github/workflows/nix-pull.yaml

```
@@ -133,6 +143,44 @@ jobs:
           crane push image.tar "${DOCKER_REGISTRY}/${BASE_IMAGE}"
           crane auth logout ${DOCKER_REGISTRY}
 
+  analyze-build-times:
+    name: Analyze Build Times
+    needs: [build, check-conditions]
+    if: needs.check-conditions.outputs.should-proceed == 'true' && needs.check-conditions.outputs.has-backend == 'true'
+    runs-on: ${{ matrix.host }}
```

actionlint (1.7.11)

[error] property "host" is not defined in object type {}

(expression)